app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -82,6 +181,18 @@ class SuggestedEditsTasksFragment : Fragment() {
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
         inflater.inflate(R.menu.menu_suggested_edits_tasks, menu)
+        var drawable: Drawable = menu.findItem(R.id.menu_help).icon
+        drawable = DrawableCompat.wrap(drawable)
+        DrawableCompat.setTint(drawable, ResourceUtil.getThemedColor(context!!, R.attr.colorAccent))
+        menu.findItem(R.id.menu_help).icon = drawable
```

Comment:
Can we set the tint in the `R.menu.menu_suggested_edits_tasks` instead of programmatically setting it up?

Reply:
iconTint is available only >API26. that's why.